runtime/karura/src/lib.rs

```diff
@@ -2613,8 +2628,8 @@ mod tests {
 	#[test]
 	fn check_call_size() {
 		assert!(
-			core::mem::size_of::<Call>() <= 260,
-			"size of Call is more than 260 bytes: some calls have too big arguments, use Box to \
+			core::mem::size_of::<Call>() <= 280,
```

Comment:
why increase this?

Reply:
The new call size is 272, so I increased it to 280.
Feels like this change is from Masters, I will confirm.

Reply:
Yes can confirm on Master branch, call size is 272

Reply:
but CI passed on master?

Reply:
yeah very strange. I'll revert and test using CI. I will also test on my windows machine. Might be build environment issue?

Reply:
could be. wasm is 32bits and native is 64bits so size of some data structure will be different

Reply:
i think the call size is OK on my windows. If this passes in CI (which master passes) it must be only an mac m1 chip thing. 
I'll keep this reverted anyway, and keep the change local so tests will pass locally.

Reply:
Can confirm on my PC's Ubunto, the call size is 256.